plugins/search-backend-module-elasticsearch/src/alpha.ts

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  coreServices,
+  createBackendModule,
+} from '@backstage/backend-plugin-api';
+import { loggerToWinstonLogger } from '@backstage/backend-common';
+import { searchEngineRegistryExtensionPoint } from '@backstage/plugin-search-backend-node/alpha';
+
+import {
+  ElasticSearchHighlightConfig,
+  ElasticSearchQueryTranslatorOptions,
+  ElasticSearchConcreteQuery,
+  ElasticSearchCustomIndexTemplate,
+  ElasticSearchCustomIndexTemplateBody,
+  ElasticSearchQueryTranslator,
+  ElasticSearchSearchEngine,
+} from './engines';
+
+/**
+ * @alpha
+ * Options for {@link searchModuleElasticsearchEngine}.
+ */
+export type SearchModuleElasticsearchEngineOptions = {
+  translator?: ElasticSearchQueryTranslator;
+  indexTemplate?: ElasticSearchCustomIndexTemplate;
```

Comment:
Just a heads up that if your alpha exports indirectly re-export stable symbols from the same package then your alpha implementation would need to reference them from the main entry point to avoid a duplicate export, e.g. `import { ElasticSearchCustomIndexTemplate } from '@backstage/plugin-search-backend-module-elasticsearch'`

Reply:
A-ha, this answers my question above. We will take a look at all re-exports across the alpha exports.